changelog.d/10115.bugfix

```diff
@@ -0,0 +1,2 @@
+Fix a bug that prevented the 'ip_range_whitelist' configuration to work for federation and identity servers.
+Notice that this change will not be applied, if your configuration still uses 'federation_ip_range_blacklist', to allow backwards-compatibility.
```

Comment:
we wouldn't normally include this level of detail in the changelog. I think you can safely omit it.
```suggestion
```

Reply:
Thank you. I thought, it might be interesting for people, who still use `federation_ip_range_blacklist`. So I preferred to add some more details, just in case.
I'll remove that line.